tonic/Cargo.toml

```diff
@@ -50,7 +50,7 @@ channel = []
 [dependencies]
 base64 = "0.21"
 bytes = "1.0"
-futures-util = {version = "0.3", default-features = false}
+pin-utils = "0.1"
```

Comment:
I think we can use https://docs.rs/tokio/latest/tokio/macro.pin.html

Reply:
For now, `tokio` has been used behind `transport` feature. To use `tokio::pin` macro, we have to use `tokio` throughout `tonic` implementations. I think `tonic` is mostly used with `tokio` so it seems not to be really matter. However, as I'm not sure whether it is appropriate to always depend on `tokio` only for using `tokio::pin`, I would like to hear your opinion.

Reply:
If we depend on tokio's default features (which should be basically empty) then it should be available and pretty light weight.

Reply:
Thanks for reviews! I'll fix to use `tokio`.

Reply:
Fixed.